.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger nightly build
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 * * * *'
+  workflow_dispatch:
+
+jobs:
+  trigger-nightly:
+    name: Push tag for nightly build
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.NIGHTLY_BUILD_GH_TOKEN }}
+      -
+        name: 'Push new tag'
+        # TODO determine nightly version based on latest minor version
```

Comment:
This is something we have to do before releasing v0.2.0. Do you want to add it to this PR or create a follow-up?

Reply:
I think it would be good to see if there's much work left in the other comments/feedback. If not, I'll try adding it. If yes, I think it would be good to merge sooner rather than later, so we can start building the nigthlies.

Reply:
Good point. It's your call how you want to proceed, as long as you pick one of those options 👍

Reply:
Opened a new issue: https://github.com/ConduitIO/conduit/issues/186.